doc/external_docs.css

```diff
@@ -0,0 +1,19 @@
+/* unvisited link */
+a:link {
+    color: #4d76ae;
```

Comment:
If you look at the `generate_docs.sh` that was added along with this change, that copies this new css file into the `target/doc` location while building the documentation.